servers/v3-proxy-api/src/graph/graphQLClient.ts

```diff
@@ -157,7 +157,7 @@ export class GraphQLClientFactory {
             !(
               error.path &&
               Array.isArray(error.path) &&
-              error.path.indexOf('recentSearches') &&
+              error.path.indexOf('recentSearches') >= 0 &&
```

Comment:
small bugfix, I don't think `recentSearches` is ever possible to be in the leftmost (zero-indexed) path but let's fix it anyway